src/components/Footer.js

```diff
@@ -137,7 +137,8 @@ const Footer = () => {
                 <a
                   className='text-blue-800 opacity-75 hover:opacity-100'
                   href={'https://github.com/drewclem/protege/issues'}
-                  target='_'
+                  target='_blank'
+                  rel='noopener noreferrer'
```

Comment:
Tightening up things. 👏🏻 I wonder why the [jsx-no-target-blank](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md) eslint rule wasn't picking this up. I believe it 's part of Create React Apps's eslint configuration.

Reply:
Just missed that. Definitely want your change here.